src/main/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackOrchestratorController.java

```diff
@@ -26,11 +26,11 @@ public CcdCallbackOrchestratorController(final AuthorisationService authorisatio
     }
 
     @RequestMapping(value = "/send", produces = APPLICATION_JSON_VALUE, method = RequestMethod.POST)
-    public ResponseEntity send(
+    public ResponseEntity<String> send(
         @RequestHeader(AuthorisationService.SERVICE_AUTHORISATION_HEADER) String serviceAuthHeader,
         @RequestBody String body) {
         authorisationService.authorise(serviceAuthHeader);
         topicPublisher.sendMessage(body);
-        return new ResponseEntity(HttpStatus.OK);
+        return new ResponseEntity<>("{}", HttpStatus.OK);
```

Comment:
Side note, was it planned to return 200 for POST message?

Reply:
It did return 201 (https://github.com/hmcts/sscs-ccd-callback-orchestrator/pull/24), however the callback from ccd didn't like it. The CCD callback would not throw an error on the screen for a 200. Ccd, now seems to require a value in the body.

Reply:
ah. thanks